app/templates/events.hbs

```diff
@@ -24,7 +22,7 @@
 {{#each event in model}}
   {{#link-to 'check-in'}}
     <div class="col-md-6">
-      <div {{bind-attr class=':panel event.track.style'}}>
+      <div {{bind-attr class=':panel event.track.style :event'}}>
```

Comment:
should be `class="panel {{event.track.style}} event"` to remove deprivation warnings